src/main/java/com/weathe_alligator/user_service/mapper/LocationMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.weathe_alligator.user_service.mapper;
+
+import com.weathe_alligator.user_service.dto.LocationDTO;
+import com.weathe_alligator.user_service.entity.Location;
+
+public class LocationMapper {
+    private LocationMapper() {
+    }
+
+    public static LocationDTO getLocationDTO(Location location) {
```

Comment:
refactor method to `public static LocationDTO toDTO(Location entity) {`
same way at userMapper